src/TwigComponent/src/Resources/doc/index.rst

```diff
@@ -360,8 +360,12 @@ need to populate, you can render it with:
     means that you can safely render the same component multiple times with
     different data because each component will be an independent instance.
 
-Computed Properties
-~~~~~~~~~~~~~~~~~~~
+Computed Methods
```

Comment:
Not sure about this change. Should we keep as _Computed Properties_? Seems to be the more common term.

Reply:
I had been wondering about this too. Yes, I think we keep it as `Computed Properties`. And you've already documented it in the way that I would have: you document using it *like* you're referencing a property: `this.products`. A method is called behind the scenes to calculate this, but it looks like a property.